media/CircleGraph/index.js

```diff
@@ -430,6 +433,9 @@ host.BrowserHost = class {
     return new Promise((resolve, reject) => {
       if (url.startsWith("vscode-webview://")) {
         const messageHandler = (event) => {
+          if(window.origin === event.origin){
+            return;
+          }
```

Comment:
(ditto)
I just applied it empty-minded. 
This one makes circle viewer not working. 
Let me know if anyone knows the right way.